tests/test_check.py

```diff
@@ -492,6 +492,21 @@ def test_exclusion_does_exclude_custom_check():
     assert issues == []
 
 
+# Issues at $.
```

Comment:
We could also put this under an `explain` test section, let me know if that is preferred.